imageloading-core/src/main/java/com/google/accompanist/imageloading/AndroidDrawablePainter.kt

```diff
@@ -75,11 +94,18 @@ class AndroidDrawablePainter(
 
         // Update the drawable's bounds to match the intrinsic size
         drawable.setBounds(0, 0, drawable.intrinsicWidth, drawable.intrinsicHeight)
+    }
 
-        // If this is animatable, start it now
-        if (drawable is Animatable && !drawable.isRunning) {
-            drawable.start()
-        }
+    internal fun onRemembered() {
```

Comment:
given that this class is not really useful without people being able to call onRemembered() on it maybe this class should be internal/private?

Reply:
Yeah agreed, done.